twoliter/embedded/Makefile.toml

```diff
@@ -1412,6 +1421,42 @@ pubsys \
 '''
 ]
 
+[tasks.fetch-variant]
+script_runner = "bash"
+script = [
+'''
+set -e
+
+export PATH="${TWOLITER_TOOLS_DIR}:${PATH}"
+
+if [ -z "${FETCH_VARIANT_LOCAL_ROOT_JSON}" ]; then
+   fetch_variant_root="./fetch-variant-root.json"
+   echo "Fetching root.json..."
+   curl -s -L -C - "${FETCH_VARIANT_ROOT_JSON}" -o "${fetch_variant_root#./}"
+   sha512sum -c <<<"${FETCH_VARIANT_ROOT_JSON_SHA512}  ${fetch_variant_root#./}"
+else
+   fetch_variant_root="${FETCH_VARIANT_LOCAL_ROOT_JSON}"
+fi
+
+if [ "${FETCH_VARIANT_UNIFIED}" == "true" ]; then
+   tuftool download \
+      "${BUILDSYS_VARIANT_DIR}" \
+      --target-name "${BUILDSYS_NAME_FRIENDLY}.img.lz4" \
```

Comment:
This doesn't work for VMware variants which have different suffixes.

Reply:
I planned on adding some VMware-specific logic in a later PR.

Reply:
Makes sense it's just calling out that this doesn't work for those variants.